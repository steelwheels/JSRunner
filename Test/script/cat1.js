/* cat1.js */

function cat(){
	var file = FileManager.open("../LICENSE", "r")
	if(file != null){
		var line
		console.log("*** File Start")
		while((line = file.getl()) != null){
			stdout.put(line)
		}
		console.log("*** File End")
	} else {
		console.error("Failed to open file\n")
	}
}

cat()
console.log("*** Bye")

