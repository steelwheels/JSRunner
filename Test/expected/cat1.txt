*** File Start
/* cat1.js */

function cat(){
	var file = File.open("../sample/cat1.js", "r")
	if(file != null){
		var line
		console.log("*** File Start\n")
		while((line = file.getl()) != null){
			stdout.put(line)
		}
		console.log("*** File End\n")
	} else {
		console.log("Failed to open file\n")
	}
}

cat()
console.log("*** Bye\n")

*** File End
*** Bye
